Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/ACL/Init.hs

```diff
@@ -54,6 +55,7 @@ buildInitReq subscriber req = do
         bapId = subscriber.subscriber_id,
         bapUri = subscriber.subscriber_url,
         maxEstimatedDistance = order.fulfillment.tags.max_estimated_distance,
+        paymentMethodInfo = mkPaymentMethodInfo <$> order.payment,
```

Comment:
@roman-bodavskiy We were facing some backward compatibility issue for BAP to BPP or BPP to BAP in beckn protocol, as sometime BAP is not released or on older version and expect something to come from BPP and vice versa.

So, in this case `order.payment` is considered as List type, what is the older type for this? Can you handle that as well?

Reply:
Not `List`, now its `Maybe` type, older was mandatory type. Agree, this is not backward compatible, because old version BPP will expect mandatory `Payment` and can't parse `null`.